ml_service/pipelines/verify_train_pipeline.py

```diff
@@ -0,0 +1,67 @@
+import os
+import sys
+import argparse
+from azureml.core import Run, Experiment, Workspace
+
+
+def main():
+
+    run = Run.get_context()
+    if (run.id.startswith('OfflineRun')):
+        from dotenv import load_dotenv
+        sys.path.append(os.path.abspath("./code/util"))  # NOQA: E402
+        from model_helper import get_model_by_tag
+        # For local development, set values in this section
+        load_dotenv()
+        workspace_name = os.environ.get("WORKSPACE_NAME")
+        experiment_name = os.environ.get("EXPERIMENT_NAME")
+        resource_group = os.environ.get("RESOURCE_GROUP")
+        subscription_id = os.environ.get("SUBSCRIPTION_ID")
+        build_id = os.environ.get('BUILD_BUILDID')
+        aml_workspace = Workspace.get(
+            name=workspace_name,
+            subscription_id=subscription_id,
+            resource_group=resource_group
+        )
+        ws = aml_workspace
+        exp = Experiment(ws, experiment_name)
+    else:
+        sys.path.append(os.path.abspath("./util"))  # NOQA: E402
+        from model_helper import get_model_by_tag
+        ws = run.experiment.workspace
+        exp = run.experiment
+
+    parser = argparse.ArgumentParser("register")
+    parser.add_argument(
+        "--build_id",
+        type=str,
+        help="The Build ID of the build triggering this pipeline run",
+    )
+    parser.add_argument(
+        "--model_name",
+        type=str,
+        help="Name of the Model"
```

Comment:
Let's put a default value here if none specified.  Also default value in the variables template.

Reply:
Great point. I've removed the argument and the script just takes the value from e.model_name.